src/test/denialofservice_tests.cpp

```diff
@@ -129,8 +130,7 @@ BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
     BOOST_CHECK(dummyNode1.fDisconnect == true);
     SetMockTime(0);
 
-    bool dummy;
-    peerLogic->FinalizeNode(dummyNode1, dummy);
+    peerLogic->FinalizeNode(dummyNode1);
```

Comment:
Just an observation: before this patch `AddrMan::Connected()` would not have been called here and it will be called after this patch. Also `PeerManager::FinalizeNode()` would try to dereference `PeerManager::m_addrman`. It is fine, as far as I can see.

Reply:
Yes, this is a slightly different ordering, but I agree that it's fine. Everything between here and the end of Finalize is just internal net_processing book-keeping.